reference/commands/consumer/install.rst

```diff
@@ -312,6 +322,24 @@ With the :command:`-o` parameters you can only define specific package options.
     You can use :ref:`profiles <profiles>` files to create predefined sets of **settings**,
     **options** and **environment variables**.
 
+conf
+----
+
```

Comment:
Add here a note that the [conf] is an experimental feature

Reply:
Done 😉